hardware_interface/include/hardware_interface/robot_hw.h

```diff
@@ -148,33 +152,72 @@ class RobotHW : public InterfaceManager
     ERROR
   };
 
-  // Return (in realtime) the state of the last doSwitch()
+  /** \brief Return (in realtime) the state of the last doSwitch() */
   virtual SwitchState switchResult() const
   {
     return DONE;
   }
 
-  // Return (in realtime) the state of the last doSwitch() for a given controller
+  /** \brief Return (in realtime) the state of the last doSwitch() for a given controller */
   virtual SwitchState switchResult(const ControllerInfo& /*controller*/) const
   {
     return DONE;
   }
+  /**\}*/
+
+  
+  /** \name Control Loop
+   * 
+   */
+   /**\{*/
 
-  /**
-   * Reads data from the robot HW
+  /** \brief Used to read data from the robot hardware (represented by its interfaces) and populate the raw data.
```

Comment:
Let's keep this shorter and in the same tense as the rest of the documentation. `Read data from the robot hardware`, for example.

Reply:
👍